src/session-managers/SecureSessionManager.ts

```diff
@@ -53,7 +56,8 @@ export class SecureSessionManager {
     // TODO: This is quite hacky. The best option would be having a regenerate method
     // on secure-session as well
     else if (this.clearSessionOnLogin && object) {
-      const currentFields = request.session.data() || {}
+      const currentData: SessionData = request.session?.data() ?? {}
+      const currentFields = currentData ?? {}
       for (const field of Object.keys(currentFields)) {
```

Comment:
I think we can remove initalizing `currentFields` and just pass `currentData` to `Object.keys(currentData)`
```suggestion
      const currentData: SessionData = request.session?.data() ?? {}
      for (const field of Object.keys(currentData)) {
```